src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -823,8 +867,79 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
     </div>
   );
 
+  const { sovereignVenueId } = useSovereignVenueId({ venueId });
+
+  const { currentVenue: venue } = useConnectCurrentVenueNG(sovereignVenueId);
+
+  const [userStatuses, setUserStatuses] = useState<UserStatus[]>([]);
+
+  const {
+    isShown: hasUserStatuses,
+    show: showUserStatuses,
+    toggle: toggleUserStatus,
+  } = useShowHide(venue?.showUserStatus);
+
+  // Because this is not using the useForm validation. The use effect needs to manually open the dropdown with user statuses.
+  useEffect(() => {
+    if (!venue) return;
+    const venueUserStatuses = venue?.userStatuses ?? [];
+
+    setUserStatuses(venueUserStatuses);
+
+    if (venue.showUserStatus) {
+      showUserStatuses();
+    }
+  }, [showUserStatuses, venue]);
+
+  const removeUserStatus = (index: number) => {
+    const statuses = [...userStatuses];
+    statuses.splice(index, 1);
+    setUserStatuses(statuses);
+  };
+
+  const addUserStatus = () =>
+    setUserStatuses([...userStatuses, { status: "", color: "#4BCC4B" }]);
```

Comment:
I suppose that we have a `theme` obj at `theme.ts`, so let's import that color from there

Reply:
Pretty sure theme is legacy tech debt related to styled components.

Reply:
But should definitely be imported from a const. Though potentially in src/settings

Reply:
I guess we can use here one of the constants from settings which were created for default statuses, can't we?

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1532/commits/0c1f9f07a5696848014d78ad2604de3c1ed2d194